pkg/linode-bs/luks.go

```diff
@@ -66,16 +66,16 @@ func (ctx *LuksContext) validate() error {
 
 	var err error
 	if ctx.VolumeName == "" {
-		err = errors.Join(err, fmt.Errorf("no volume name provided"))
+		err = errors.Join(err, errors.New("no volume name provided"))
 	}
 	if ctx.EncryptionKey == "" {
-		err = errors.Join(err, fmt.Errorf("no encryption key provided"))
+		err = errors.Join(err, errors.New("no encryption key provided"))
 	}
 	if ctx.EncryptionCipher == "" {
-		err = errors.Join(err, fmt.Errorf("no encryption cipher provided"))
+		err = errors.Join(err, errors.New("no encryption cipher provided"))
 	}
 	if ctx.EncryptionKeySize == "" {
-		err = errors.Join(err, fmt.Errorf("no encryption key size provided"))
+		err = errors.Join(err, errors.New("no encryption key size provided"))
```

Comment:
Could you define those errors as constants somewhere, and just reference them here by name?

```suggestion
		err = errors.Join(err, ErrNoVolumeName)
	}
	if ctx.EncryptionKey == "" {
		err = errors.Join(err, ErrNoEncryptionKey)
	}
	if ctx.EncryptionCipher == "" {
		err = errors.Join(err, ErrNoEncryptionCipher)
	}
	if ctx.EncryptionKeySize == "" {
		err = errors.Join(err, ErrNoEncryptionKeySize)
```

Reply:
I don't think that it makes sense to create consts for these as they're only being used once. I don't see the value add in the indirection if they're a single instance.